types/module/module.go

```diff
@@ -261,6 +262,11 @@ func (GenesisOnlyAppModule) EndBlock(sdk.Context) ([]abci.ValidatorUpdate, error
 	return []abci.ValidatorUpdate{}, nil
 }
 
+// ConsensusParamsGetter is an interface to retrieve consensus parameters for a given context.
+type ConsensusParamsGetter interface {
+	GetConsensusParams(ctx sdk.Context) tmproto.ConsensusParams
```

Comment:
what if we just return an `any` here and then type cast? Then we can avoid the CometBFT dependency.

Reply:
or can we add a separate package to avoid import cometbft type directly